plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiUtil.java

```diff
@@ -152,22 +151,13 @@ public static HoodieTableMetaClient buildTableMetaClient(
                 .build();
     }
 
-    public static Schema constructSchema(List<String> columnNames, List<HiveType> columnTypes, boolean withMetaColumns)
```

Comment:
Is there still a case where the caller can call `withMetaColumns=true`?

Reply:
All usages referencing this function was passing `withMetaColumns=false`, that's why i removed it.

Could not find any usages with `withMetaColumns=true`

Reply:
Got it